subsys/bluetooth/mesh/light_ctl_cli.c

```diff
@@ -214,8 +214,14 @@ static int get_msg(struct bt_mesh_light_ctl_cli *cli,
 	BT_MESH_MODEL_BUF_DEFINE(msg, opcode, BT_MESH_LIGHT_CTL_MSG_LEN_GET);
 	bt_mesh_model_msg_init(&msg, opcode);
 
-	return model_ackd_send(cli->model, ctx, &msg,
-			       rsp ? &cli->ack_ctx : NULL, ret_opcode, rsp);
+	struct bt_mesh_msg_rsp_ctx rsp_ctx = {
+		.ack = &cli->ack_ctx,
+		.op = ret_opcode,
+		.user_data = rsp,
+		.timeout = model_ackd_timeout_get(cli->model, ctx),
+	};
+
+	return bt_mesh_msg_ackd_send(cli->model, ctx, &buf, rsp ? &rsp_ctx : NULL);
```

Comment:
```suggestion
	return bt_mesh_msg_ackd_send(cli->model, ctx, &msg, rsp ? &rsp_ctx : NULL);
```
? Applies several places in this file